cli/src/main/scala/aqua/AppOpts.scala

```diff
@@ -193,7 +184,16 @@ object AppOpts {
 
   val scriptOpt: Opts[Boolean] =
     Opts
-      .flag("scheduled", "Generate air code for script storage. Without error handling wrappers and hops on relay. Will ignore other options")
+      .flag(
+        "scheduled",
+        "Generate air code for script storage. Without error handling wrappers and hops on relay. Will ignore other options"
+      )
+      .map(_ => true)
+      .withDefault(false)
+
+  val noPrelude: Opts[Boolean] =
+    Opts
+      .flag("no-prelude", "Don't use default aqua import paths")
```

Comment:
Let's remove that flag altogether. It seems that there's no use-case for it.

There's a consideration, however, about priorities between local `aqua-lib` and a global one. Local `aqua-lib` (from node_modules in the current dir) should always be used when available, and global one ignored.

In other words, we need to make sure that the following scenario works:
1. Install global aqua via `npm i -g`
2. Install local aqua via `npm --save-dev`, so it is in package.json
3. Install local `aqua-lib@v0.2.0` via `npm --save-dev @fluencelabs/aqua-lib@v0.2.0`, so it is in package.json
4. Write simple aqua script that uses `Peer.timeout` from `@fluencelabs/aqua-lib`
5. Compile with global aqua, should fail cuz `Peer.timeout` isn't available in `0.2.0`
6. Compile with local aqua, should fail as well

In this scenario, we check that local aqua-lib of version 0.2.0 takes precedence of global aqua-lib of version 0.2.1. Usually, it's gonna be the other way around: global aqua-lib would be older than the local one.